admin/ajax.php

```diff
@@ -359,3 +359,6 @@ function wpseo_add_fb_admin() {
 }
 
 add_action( 'wp_ajax_wpseo_add_fb_admin', 'wpseo_add_fb_admin' );
+
+// Crawl Issue Manager AJAX hooks.
+new WPSEO_GWT_Ajax;
```

Comment:
If you create the object this way the actions and filters it creates are not overridable by other plugins/themes. Easiest way to fix is using a global/static and a function to retrieve that global/static.